studio/schemas/fields/media.ts

```diff
@@ -38,6 +48,31 @@ const image = defineField({
   fields: [imageAltField],
 });
 
+export const internationalizedImage = defineField({
```

Comment:
I think it is a bit strange to name it internalinalizedImage

Reply:
Yea, I guess the other alternative I had was `imageWithInternationalizedAlt`, which was not great either 😅 

The main point was to not replace the original image, to not break other parts of the project. So do you have a suggestion for a different name?

Reply:
Yeah, I agree that InternationalizedImage is better. I can't think of a better name, so I guess it's fine.